src/wallet/wallet.cpp

```diff
@@ -2992,8 +3017,10 @@ std::string CWallet::GetWalletHelpString(bool showDebug)
     return strUsage;
 }
 
-CWallet* CWallet::InitLoadWallet(bool fDisableWallet, const std::string& strWalletFile, std::string& warningString, std::string& errorString)
+bool CWallet::InitLoadWallet()
 {
+    std::string strWalletFile = GetArg("-wallet", DEFAULT_WALLET_DAT);
+
```

Comment:
Nit:  this local variable has the same name as the class member variable (I guess it is not a problem here since the compiler knows it can't access the class member from a static method?)  Not a big deal but maybe want to give it a different name.